src/public/meeting.ts

```diff
@@ -8,6 +8,24 @@ export namespace meeting {
    * Data structure to represent a meeting details.
    */
   export interface IMeetingDetails {
+    /**
+     * details object
+     */
+    details: IDetails;
```

Comment:
Seems a little redundant to have an `IDetails` object within an `IMeetingDetails` object. Should we maybe rename the outer interface to `IMeetingProperties` and change the SDK API name to `getMeetingProperties`? What is the reason for creating this additional nesting?

Reply:
@ydogandjiev we are working on making our response like this 
`{​​​​
  "details": {​​​​
     "id": " MCMxOTptZWV0aW5nX05EUXhNemcxWWpVdE1HSXpOQzAwWWpjMUxXRm1ZV1l0WXprMU1HWTJNVE13TmpFMEB0aHJlYWQudjIjMA==",
    "scheduledStartTime": "2020-08-21T02:30:00+00:00",
    "scheduledEndTime": "2020-08-21T03:00:00+00:00",
    "JoinUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_NDQxMzg1YjUtMGIzNC00Yjc1LWFmYWYtYzk1MGY2MTMwNjE0%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%226aebbad0-e5a5-424a-834a-20fb051f3c1a%22%7d",
    "title": "Teams All Hands",
    "type": "Scheduled"
  }​​​​,
  "conversation": {​​​​
    "id": "19:meeting_NDQxMzg1YjUtMGIzNC00Yjc1LWFmYWYtYzk1MGY2MTMwNjE0@thread.v2"
  }​​​​,
  "organizer": {​​​​
    "id": "29:1JKiJGPAX9TTxtGxhVo0wLx_zwzo-gG8Z-X03306vBwi9p-xMTEbDXsT6KH7-0kkTS8cD-2zkrsoV6f5WJ6_aYw",
    "tenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47"
  }​​​​
}​​​​`

Reply:
As discussed in the chat thread, I am marking these comments as resolved. @ydogandjiev please let me know if you have any other feedback :)